internal/internal_worker_base.go

```diff
@@ -252,6 +252,7 @@ func (bw *baseWorker) runPoller() {
 		case <-bw.shutdownCh:
 			return
 		case <-bw.pollerRequestCh:
+			bw.metricsScope.Gauge(metrics.PollerRemainingBuffer).Update(float64(len(bw.pollerRequestCh)))
```

Comment:
as mentioned above we can emit two gauge.
```suggestion
			bw.metricsScope.Gauge(metrics.ConcurrentTaskQuota).Update(float64(cap(bw.pollerRequestCh)))
                         bw.metricsScope.Gauge(metrics.ConcurrentTaskRunning).Update(float64(cap(bw.pollerRequestCh) - len(bw.pollerRequestCh)))
```